content/projects/oop/person/_index.md

```diff
@@ -17,6 +17,66 @@ tags:
 title: Person
 ---
 
+## Set up your environment
+
+### JavaScript
+
+Your directory structure should look like this.
+
+```
+    >node_modules    <---- make sure this is in your .gitignore
+    >spec
```

Comment:
Can you please use a  consistent style for showing file hierarchies? 

I know things aren't always consistent in other places in the syllabus, but we should always try to improve things as we go.

The way you showed the Python and Java stuff is nicest